index.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  Bridge: require('./lib/bridge.js'),
-  Peer: require('./lib/peer.js'),
-  PeerGroup: require('./lib/peerGroup.js')
+  Bridge: require('./src/bridge.js'),
+  Peer: require('./src/peer.js'),
+  PeerGroup: require('./src/peerGroup.js')
```

Comment:
:-1: This breaks the build setup. We use `lib` because it is a symlink to `src` while developing, and contains the ES5 build after a `npm run build` or `npm publish`.